### ISSUE TITLE
New english-gb messes up apostrophees 

### ISSUE THREAD

**Issue by Marcool04** [r-stein/sublime-text-latex-smart-quotes#9]
I'm so so sorry to bother you with this, but the pull request I authored a few weeks ago and that you accepted actually has an issue with apostrophes...

I commented on the pull request (https://github.com/r-stein/sublime-text-latex-smart-quotes/pull/8#issuecomment-1377384152) but since it is closes here is my message again:

When using the 'english-gb' setting, attempting to insert an apostrophe becomes muddled.

![screencast](https://user-images.githubusercontent.com/15948152/211581203-8e33677e-6a60-492f-a2fb-939ce4c84703.gif)

Basically, the keys for apostrophe and quote are "inverted", when attempting to insert an apostrophe to write something like "it's" or "won't"...

I had a quick look at the code, but I couldn't seem to figure out where this is detected. Sorry to be bugging you about a problem I created, with a pull request for a feature I'm possibly the only one to use 😔
